akka-persistence/src/main/scala/akka/persistence/Plugin.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence
+
+import java.util.concurrent.atomic.AtomicReference
+
+import akka.actor.{ ExtendedActorSystem, Extension, ExtensionId }
+import akka.annotation.InternalApi
+import akka.event.Logging
+import akka.persistence.Plugin.PluginHolder
+import com.typesafe.config.Config
+
+import scala.annotation.tailrec
+import scala.reflect.ClassTag
+import scala.util.Failure
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[akka] object Plugin {
```

Comment:
PersistencePlugin to avoid popping up so much in completion?

Reply:
Yep. Will rename